HISTORY.rst

```diff
@@ -2,6 +2,29 @@
 History
 =======
 
+3.44.0 (2021-09-16)
+-------------------
+
+Changes
+
+- CumulusCI uses package version Ids from 1GP releases wherever available, reducing the need to install 1GP packages in an org to build 2GP dependencies. (#2832)
+- We added ``metadata_package_id`` and ``version_id`` options to allow passing ``MetadataPackage`` (prefix 033) and ``MetadataPackageVersion`` (prefix 04t) IDs to the push upgrade tasks. (#2837)
+- ``cci flow info`` will now output all flow options defined. (#2845)
+- We improved error messages for incorrect data mapping files (#2831).
+- The ``snowfakery`` task supports ``loading_rules`` to be supplied in addition to implied by filename (#2861)
+- The ``snowfakery`` task supports recipe options being supplied to recipes as the older ``generate_and_load_from_yaml`` task did (#2861).
```

Comment:
can we at least mention the name of the option so users know what to search for for more information?